src/python/grpcio/grpc/_cython/_cygrpc/completion_queue.pyx

```diff
@@ -74,7 +68,41 @@ cdef class CompletionQueue:
     with self.poll_condition:
       self.is_polling = False
       self.poll_condition.notify()
+    return self.interpret_event(event)
+
+  def pluck(self, records.OperationTag tag, records.Timespec deadline=None):
+    # Plucking a 'None' tag is equivalent to passing control to GRPC core until
+    # the deadline.
+    # TODO(atash) there's currently no way of getting a tag to pass to this
```

Comment:
It's more that this is a function that was created specifically to allow for one to pass control to GRPC s.t. it may do work without risking the receipt of an event that one cannot handle, but it's basically a wrapper around grpc_completion_queue_pluck and thus is named that way. It cannot be used (fruitfully) in any way, currently, other than by passing None.